sdk/mobile-center/src/test/java/com/microsoft/azure/mobile/channel/AbstractDefaultChannelTest.java

```diff
@@ -0,0 +1,119 @@
+package com.microsoft.azure.mobile.channel;
+
+import android.content.Context;
+import android.os.Handler;
+import android.os.HandlerThread;
+import android.os.Looper;
+
+import com.microsoft.azure.mobile.ingestion.ServiceCallback;
+import com.microsoft.azure.mobile.ingestion.models.Device;
+import com.microsoft.azure.mobile.ingestion.models.Log;
+import com.microsoft.azure.mobile.persistence.DatabasePersistenceAsync;
+import com.microsoft.azure.mobile.utils.DeviceInfoHelper;
+import com.microsoft.azure.mobile.utils.IdHelper;
+import com.microsoft.azure.mobile.utils.MobileCenterLog;
+import com.microsoft.azure.mobile.utils.UUIDUtils;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.mockito.Mock;
+import org.mockito.internal.stubbing.answers.Returns;
+import org.mockito.invocation.InvocationOnMock;
+import org.mockito.stubbing.Answer;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.rule.PowerMockRule;
+
+import java.util.ArrayList;
+
+import static com.microsoft.azure.mobile.persistence.DatabasePersistenceAsync.THREAD_NAME;
+import static org.mockito.Matchers.any;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.powermock.api.mockito.PowerMockito.mockStatic;
+import static org.powermock.api.mockito.PowerMockito.whenNew;
+
+@SuppressWarnings("WeakerAccess")
+@PrepareForTest({DefaultChannel.class, IdHelper.class, DeviceInfoHelper.class, DatabasePersistenceAsync.class, MobileCenterLog.class})
+public class AbstractDefaultChannelTest {
+
+    static final String TEST_GROUP = "group_test";
```

Comment:
It won't complain (I tried). We can change that later if we have any sub-classes in other packages.